src/Athena.Importer/data/course/mthm280.json

```diff
@@ -0,0 +1,10 @@
+{
+  "id": "8588ca76-eaa0-43d0-bead-ad1c3374a881",
+  "name": "MTHM  280 - LINEAR ALGEBRA",
```

Comment:
Strip extra space from name (Between `MTHM` and `280`)

Reply:
DONE